### ISSUE TITLE
Meet Web Content Accessibility Guidelines (WCAG) for the Live Chat client

### ISSUE THREAD

**Issue by koudshoorn** [chatwoot/chatwoot#1940]
**Is your enhancement request related to a problem? Please describe.**
The current chat client that is used for the website channels is not yet accessible for people with certain disabilities (like blindness). Accommodating these people is a requirement that we're trying to uphold for our website. With a number of minor changes, we believe this accessibility requirement could be met.

**Describe the solution you'd like**
The description of WCAG is quite extensive, but we believe that by focusing on 3 things a large portion of the guidelines could be met:
1. The chat client should be controllable without the use of a mouse
2. Controls and inputs should be recognizable by a screen-reader and have appropriate labels attached that can be read aloud. (e.g. icon buttons have a text alternative)
3. Contrast and button sizes should be increased. (Contrast was already mentioned in #1346 and strongly relates to this issue)

**Additional context**
We're already working on making the appropriate changes and would like to create a PR in the short future to solve this issue.

* The full description of WCAG can be found [here](https://www.w3.org/TR/WCAG21/)
* Contrast can be measured using an online tool such as [contrast-ratio](https://contrast-ratio.com/) and should aim for a ratio of at least 4.5
* Buttons should usually be at least 44px by 44px, though there are exceptions
